tests/object_property_ignore/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -0,0 +1,665 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`ignore.js - {"trailingComma":"all"} format 1`] = `
+====================================options=====================================
+parsers: ["babel", "flow", "typescript"]
+printWidth: 80
+trailingComma: "all"
+                                                                                | printWidth
+=====================================input======================================
+foo = {
+  // prettier-ignore
+  bar:            1,
+}
+
+foo = {
+  _: '',
+  // prettier-ignore
+  bar:            1,
+}
+
+/* comments */
+foo = {
+  _: '',
+  // prettier-ignore
+  bar:            1,         // comment
+}
+
+foo = {
+  _: '',
+  // prettier-ignore
+  bar:            1,         /* comment */
+}
+
+foo = {
+  _: '',
+  // prettier-ignore
+  bar:            /* comment */          1,
+}
+
+/* RestElement */
+foo = {
+  _: '',
+  // prettier-ignore
+  ...bar,
+}
+
+// Nested
+foo = {
+  baz: {
+  // prettier-ignore
+  foo: [1, 2,    3]
+},
+  // prettier-ignore
+  bar:            1,
+}
+
+=====================================output=====================================
+foo = {
+  // prettier-ignore
+  bar:            1,
+};
+
+foo = {
+  _: "",
+  // prettier-ignore
+  bar:            1,
+};
+
+/* comments */
+foo = {
+  _: "",
+  // prettier-ignore
+  bar:            1, // comment
+};
+
+foo = {
+  _: "",
+  // prettier-ignore
+  bar:            1 /* comment */,
```

Comment:
This comment is move from after `,` to before `,`. [Array do the same](https://deploy-preview-8111--prettier.netlify.app/playground/#N4Igxg9gdgLgprEAuEBDABAXnQbQDpTroD0x6ADgE5wwwCWclAtHQOZQTUFEBMANEUFCixAFTpIAW0kIY6UcQIBdEHxARy9aAGdkoVJUoQA7gAUDCXSlQAbY6gCeutQCNKqMAGsaAZXIe6KFZkGEoAVzg1OEkXOAATOPiAGVQgsNRWOAAxTklUWkDg6zCYCFUQAAsYSRsAdQq6eG1-MDgfS0a6ADdGh2RwbWcQQO1GGFN3VjzkADNbUbUAK20ADwAhdy9fVBkkwLhZ+ciQZZWfQps4AEUwiHhDmwWQf0pRyn6XVFibcqpAmFqdDiMAqyAAHAAGNRUCCjWrucj9KhwN5dA5qACOt3gEw0VjQ2iYUDg8Xi5WoWLo1AmGWmSDmj2Oo0kdBC4SZF2u2IO9KOahgX0BwNBSH4IFCqDoNkKAGEINJUP0UQBWcphUYAFS+VgZTy6EQAklBErAfGBKHRNABBY0+GAOS4PUYAX2dQA)